Text selection UI matches behavior on Android.

mpcomplete opened the pull request:
- Handles appear with tap or long press.
- Toolbar appears with long press on text, or tap on handle.
- Correct toolbar items shown depending on context.

eseidelGoogle commented on packages/flutter/lib/src/rendering/editable_line.dart:
```diff
@@ -116,6 +116,7 @@ class RenderEditableLine extends RenderBox {
   set paintOffset(Offset value) {
     if (_paintOffset == value)
       return;
+    print('paintOffset changed');
```
Did you mean to leave this in?

mpcomplete replied:
Nope!

Hixie commented:
I'm not super familiar with the underlying code, but LGTM, modulo travis anger.

eseidelGoogle commented:
This now contains commits you didn't intend.  I think you want to rebase.  possibly `git rebase -i` to re-order, squash, etc.

mpcomplete committed:
Text selection UI matches behavior on Android.

- Handles appear with tap or long press.
- Toolbar appears with long press on text, or tap on handle.
- Correct toolbar items shown depending on context.
